src/App.tsx

```diff
@@ -1,26 +1,8 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => {
```

Comment:
I would recommend sticking to explicitly defining your functional components with the `function` keyword as opposed to defining them with arrow functions, to improve the readability in the long run, similar to how they are defined in the React docs.

https://reactjs.org/docs/components-and-props.html

Reply:
ok sir i will change @DangaRanga sir